renderer.php

```diff
@@ -767,11 +770,11 @@ protected function post_walker($cm, $discussion, $posts, $parent, $canreply, &$c
                     if ($post->nestedreplycount > 2 && $depth == 0) {
                         // Adding collapsable elements
                         $output .= "<div id=".$post->id." class='posts-collapse-container collapse'>";
-                        $output .= $this->post_walker($cm, $discussion, $posts, $post, $canreply, $count, ($depth + 1));
+                        $output .= $this->post_walker($cm, $discussion, $firstreplies, $post, $canreply, $count, ($depth + 1));
                         $output .= "<a class='posts-collapse-toggle collapse-bottom' data-toggle='collapse' data-target='#".$post->id."'></i></a>";
                         $output .= "</div>" ;
                     } else {
-                        $output .= $this->post_walker($cm, $discussion, $posts, $post, $canreply, $count, ($depth + 1));
+                        $output .= $this->post_walker($cm, $discussion, $firstreplies, $post, $canreply, $count, ($depth + 1));
                     }
```

Comment:
Make a not to include revising this section of code in the next ticket or we need to create a new ticket. This section was used to build out the nested children for posts which is not needed anymore. Meaning this section could potentially be removed.

Reply:
As discussed, this will be handled in next ticket